doc/source/whatsnew/v0.25.0.rst

```diff
@@ -106,6 +106,43 @@ the output will truncate, if it's wider than :attr:`options.display.width`
 (default: 80 characters).
 
 
+.. _whatsnew_0250.enhancements.json_normalize_with_max_level:
+
+Json normalize with max_level param support
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+:function:`json_normalize` normalizes the provided input dict to all
+nested levels. ``max_level`` parameter provides more control over what
```

Comment:
The new ``max_level`` parameter provides more control over which level to end normalization (....)